classes/02class/exercises/c04-iam01/mendoncaj/cli_commands.txt

```diff
@@ -0,0 +1,102 @@
+> aws iam create-user --user-name user_readonly_S3
+
+**output**:
+```
+{
+    "User": {
+        "Path": "/",
+        "UserName": "user_readonly_S3",
+        "UserId": "AIDAR6HBDI4EW3NAA7CQ3",
+        "Arn": "arn:aws:iam::133616060169:user/user_readonly_S3",
+        "CreateDate": "2020-07-03T02:58:33Z"
+    }
+}
+```
+
+> aws iam create-policy \
+--policy-name S3readOnly \
+--policy-document file://~/aws-policies/user_readonly_S3-policy.json
+
+```
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "s3:Get*",
+        "s3:List*"
+      ],
+      "Effect": "Allow",
+      "Resource": [
+        "arn:aws:s3:::*",
+        "arn:aws:s3:::*/*"
```

Comment:
The first line already indicates "all S3 buckets and objects" so you don't need the second line.
You should use slash(`/`) when you would like to specify objects from a bucket.

Ex: `arn:aws:s3:::devopsacademy/*` will allow access to all objects under `devopsacademy` bucket only.

Follow [this link](https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-arn-format.html) for more details.